### PR TITLE
Styling changes to the CC Libraries bar top

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
- Added an underline to "My Library" so that it signifies it is a datalist
Regarding #2108 

- Took away the bottom border for the title bar because it was looking too crowded with both the underline and the bottom border. Got designs from @davealonzo 

- Changed the height of the title to 5rem as per the design. Should we choose to add the bottom border again and take away the underline, the height of the top bar will still be the correct size. https://trello.com/c/jRlxiRui/491-design

### CODE REVIEW

**Comment by shaoshing** [comment]
Thanks @DivyaPrabhakar . By removing the bottom border, It seems like unbalanced between the top border and list underline (is it just me?). We might have to also think about the design when creating / renaming a library. You can rebase master to import these new features into your branch. 

screen shot 2015-08-21 at 9 23 54 am

screen shot 2015-08-21 at 9 36 46 am

**Comment by designedbyscience** [comment]
Can we just kill both of those hairlines? We are moving away from the hairlines in general. Maybe make the following 3 buttons have even space to that drop down (rather than align right?)

**Comment by davealonzo** [comment]
@designedbyscience We can certainly can kill the hairline below the "LIBRARIES" title to match the Layers panel. But we should keep the blue one that invokes for create or rename library workflows

**Comment by DivyaPrabhakar** [comment]
@davealonzo @designedbyscience @shaoshing  

So the consensus here is to:

1. Kill the top and bottom hairlines, and only have the underline that indicates a datalist

2. Add the underline to the bottom of My Library datalist 
alt text

3. About the evenly spacing the icons part, they are evenly spaced already because the width of the data list is pretty long. You can see that when we add the underline -- the underline goes all the way to the Share icon. Should I shorten the datalist width and then spread out the icons?

**Comment by volfied** [comment]
@DivyaPrabhakar in case you're waiting on this, this PR needs rebase.

**Comment by DivyaPrabhakar** [comment]
@davealonzo @designedbyscience @shaoshing Once I get a confirmation and answer on the comment above about the final changes, I will make the changes, rebase and have these changes be merged.

File: src/style/sections/libraries/libraries-section.less [hunk 1]
```diff
@@ -90,17 +90,24 @@
...
+        flex: none;
+        height: 5rem;
+        margin: 0 2rem;
+        align-items: center;
+        
         .drop-down {
             flex-grow: 1;
             border: 0;
-
+           border-bottom: @hairline solid fade(@color-subsection-rule, 60%);
```
Comment (iwehrman) [inline src/style/sections/libraries/libraries-section.less]: You appear to be missing exactly one space here :space_invader:

**Comment by iwehrman** [comment]
Should the datalist text be flush left here? 
flush

**Comment by DivyaPrabhakar** [comment]
Talked to @davealonzo about how the UI should look regarding @iwehrman's question about datalist alignments. He approved the changes that I made which now should look like this: 
 
alt text

- Removed hairlines
- Datalist flush left
- Extended width of datalist to reach icons
- Icons are flush right 
- Underlined the datalist

alt text

- Added padding to the bottom of the dropdown, under "Delete 'My Library'"

Changes made, back to you @iwehrman.

File: src/style/sections/libraries/libraries-section.less [hunk 2]
```diff
@@ -88,19 +88,27 @@
...
+        height: 5rem;
+        margin: 0 2rem;
+        align-items: center;
+        
         .drop-down {
             flex-grow: 1;
             border: 0;
-
+            width: 18.4rem;
+            border-bottom: @hairline solid fade(@color-subsection-rule, 60%);
```
Comment (iwehrman) [inline src/style/sections/libraries/libraries-section.less]: Should we be `fade`ing colors here? Is that kosher? Is there a better way that will make our lives easier later when trying to maintain different color themes? CC @placegraphichere

Comment (placegraphichere) [inline src/style/sections/libraries/libraries-section.less]: lets not do this. I think @underline should work for this.

**Comment by iwehrman** [comment]
Just one final question and then we're good to go.

**Comment by iwehrman** [comment]
LGTM, thanks!

**Pull request merged**